### PR TITLE
chore: redesign send screen

### PR DESCRIPTION
**Pull request by rcresnik**

### CODE REVIEW

File: brd-ios/breadwallet/src/ViewControllers/FeeSelector.swift [hunk 1]
```diff
@@ -113,13 +113,28 @@ class FeeSelector: UIView {
...
 
-        control.selectedSegmentIndex = 1
-        control.tintColor = .primaryButton
         clipsToBounds = true
+        setupSegmentControl()
+    }
+    
+    private func setupSegmentControl() {
+        control.selectedSegmentIndex = 1
+        control.tintColor = .yellow
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ViewControllers/FeeSelector.swift]: 👀

File: brd-ios/breadwallet/src/ViewControllers/PinPadViewController.swift [hunk 2]
```diff
@@ -115,12 +115,11 @@ class PinPadViewController: UICollectionViewController {
         
         switch style {
         case .white:
+            collectionView?.backgroundColor = .white
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ViewControllers/PinPadViewController.swift]: 👀

**Review by dangelovska — DISMISSED** [review]

File: brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift [hunk 3]
```diff
@@ -36,9 +36,11 @@ class RootNavigationController: UINavigationController, UINavigationControllerDe
             tintColor = LightColors.Background.one
             
         case is AccountViewController,
-            is HomeScreenViewController:
+            is HomeScreenViewController,
+            is KYCCameraViewController,
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift]: please, remove this line. this is being handled above on line 34

File: brd-ios/breadwallet/src/Redesign/Core/Constants/CornerRadius.swift [hunk 4]
```diff
@@ -18,7 +18,7 @@ enum CornerRadius: CGFloat {
     case common = 12
     /// 15 point radius
     case medium = 15
-    // 24 point radius
+    /// 24 point radius
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Core/Constants/CornerRadius.swift]: 25

**Review by kenan-mamedoff — APPROVED** [review]

**Review by dangelovska — APPROVED** [review]

**Pull request merged**